storage/innobase/fil/fil0crypt.cc

```diff
@@ -541,11 +569,16 @@ static fil_space_crypt_t *fil_space_read_crypt_data_v2(
 
   ut_ad(key_id != (uint)(~0));
 
+  char uuid[ENCRYPTION_SERVER_UUID_LEN];
+  memset(uuid, 0, ENCRYPTION_SERVER_UUID_LEN);
+  memcpy(uuid, page + offset + bytes_read, ENCRYPTION_SERVER_UUID_LEN);
+  bytes_read += ENCRYPTION_SERVER_UUID_LEN;
+
```

Comment:
Here can we assert that if crypt_schem is encrypted, uuid shouldn't be empty. (This is the reader part)
I verfied that we assert the same during write to page 0 (writer part)

Reply:
I have added the assert